boards/uz7ev_evcc/nvme/uz7ev_evcc_factest.xdc

```diff
@@ -0,0 +1,366 @@
+# ----------------------------------------------------------------------------

```

Comment:
Correct this file is not used by a Vivado project, but it is in the repo because it is a good reference because it shows the constraints for the defined interfaces.

Reply:
Agreed, but there is already one uz7ev_evcc_factest.xdc in uz7ev_evcc/base/
Do you want to keep both?

Reply:
Good catch!  No need to keep both.  Will remove from nvme folder.